### DISCUSSION TITLE
FluentDivider prerendering error

### DISCUSSION THREAD

**Discussion by geowong2022** [microsoft/fluentui-blazor#910] [General]
I hit a problem today after upgrading to v3.2 in my Blazor Server project. Previously all fine when using v3.0.1.

Error:
InvalidOperationException: JavaScript interop calls cannot be issued at this time. This is because the component is being statically rendered. When prerendering is enabled, JavaScript interop calls can only be performed during the OnAfterRenderAsync lifecycle method.

From the detail, it seems like my FluentDivider causes me problem. If I remove it, then the page is render properly.

In my _Host.cshtml,
<component type="typeof(App)" render-mode="ServerPrerendered" param-UserTicket="@result.userTicket" />

Thanks for helping me in advance. :)

**Comment by vnbaaij** [comment]
Hi,

We added a script to correct a behavior in the `FluentDivider`. Unfortunately we made a mistake there. Will be corrected in the next update.

**Reply by geowong2022** [reply]
Thanks for your help.
